<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OknoParametryTestu</class>
 <widget class="QDialog" name="OknoParametryTestu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>653</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1200</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Parametry testu</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frError">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="errorLab">
        <property name="text">
         <string>Dane poprawne</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_dane">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="3">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Wilgotność</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Ciśnienie</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="osobaWykonujaca"/>
      </item>
      <item row="3" column="5">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>[mbar]</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Rozpoczęto</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLineEdit" name="wilgotnosc"/>
      </item>
      <item row="3" column="4">
       <widget class="QLineEdit" name="cisnienie"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="ltestname">
        <property name="text">
         <string>Nazwa testu</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="dataRozpoczecia">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>[%]</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="5">
       <widget class="QLineEdit" name="testName">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>&lt;html&gt;&lt;body&gt;[&amp;deg; C]&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Osoba wykonująca test</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLineEdit" name="temperatura"/>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Temperatura</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Uwagi</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="5">
       <widget class="QTextEdit" name="uwagi"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameSpec">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_niewspolosiowosc">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="2" column="2">
          <widget class="QLabel" name="pionOdbiornik">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Oś pozioma</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="etReceiver">
           <property name="text">
            <string>Odbiornik</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Oś pionowa</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="poziomNadajnik">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="poziomOdbiornik">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="pionNadajnik">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="etTransmitter">
           <property name="text">
            <string>Nadajnik</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_powtarzalnosc">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="1">
          <widget class="QLineEdit" name="powtarzalnosc_czas">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="maxLength">
            <number>8</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>72 godziny (3 dni)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>s [od 900 (15min) do 3600 (60 min)]</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Czas pomiędzy trzecią a czwartą próbą</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Czas pomiędzy pierwszymi trzema próbami</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_dlugoscDrogiOptycznej">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="1" column="2">
          <widget class="QLabel" name="lunitrozmin">
           <property name="text">
            <string>m</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="rozstawienieMinimalne">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="lrozstawieniemaksymalne">
           <property name="text">
            <string>Rozstawienie maksymalne</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="6">
          <widget class="QLabel" name="lunitrozmax">
           <property name="text">
            <string>m</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lrozstawienieminimalne">
           <property name="text">
            <string>Rozstawienie minimalne</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLabel" name="rozstawienieMaksymalne">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_Napiecie">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="4">
          <widget class="QLabel" name="eMaksymalneNapiecie">
           <property name="text">
            <string>Maksymalne napięcie zasilania czujki</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="eMinimalneNapiecie">
           <property name="text">
            <string>Minimalne napięcie zasilania czujki</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="minVolt"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>V</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLineEdit" name="maxVolt"/>
         </item>
         <item row="0" column="6">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>V</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_czujka">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Wybór czujki</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="eReceiver">
        <property name="text">
         <string>Odbiornik</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="4">
       <widget class="QLabel" name="lUwagaWyborCzujek">
        <property name="text">
         <string>Uwaga.&lt;dla testu 0 / odtwarzalnosc&gt; , [Zamontuj kolejna czujkę z serii] / &lt;Dla pozostalych testow&gt; [&quot;Wybierz czujke zgdonie z normą : xx&quot;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cbCzujka"/>
      </item>
      <item row="1" column="3">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="eTransmitter">
        <property name="text">
         <string>Nadajnik</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="typTransmitter">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLineEdit" name="typReceiver">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>349</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbDalej">
        <property name="text">
         <string>Dalej</string>
        </property>
        <property name="icon">
         <iconset theme="media-playback-start">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbPrzerwij">
        <property name="text">
         <string>Przerwij</string>
        </property>
        <property name="icon">
         <iconset theme="media-playback-stop">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>testName</tabstop>
  <tabstop>osobaWykonujaca</tabstop>
  <tabstop>dataRozpoczecia</tabstop>
  <tabstop>temperatura</tabstop>
  <tabstop>wilgotnosc</tabstop>
  <tabstop>cisnienie</tabstop>
  <tabstop>uwagi</tabstop>
  <tabstop>powtarzalnosc_czas</tabstop>
  <tabstop>minVolt</tabstop>
  <tabstop>maxVolt</tabstop>
  <tabstop>cbCzujka</tabstop>
  <tabstop>typReceiver</tabstop>
  <tabstop>typTransmitter</tabstop>
  <tabstop>pbDalej</tabstop>
  <tabstop>pbPrzerwij</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
